src/RealmSettingsPage.tsx

```diff
@@ -63,7 +64,11 @@ const RealmSettingsForm = ({
         <div className="d-flex flex-column flex-md-row">
           <Button
             variant="danger"
-            disabled={!canSubmit || isUpdatingSettings}
+            disabled={
+              !canSubmit ||
+              isUpdatingSettings ||
+              !astarte.token?.can('realmManagement', 'GET', '/config')
```

Comment:
It's better to directly hide the button instead of displaying it disabled.

Also, the correct permission to check is:
```tsx
astarte.token?.can('realmManagement', 'PUT', '/config/auth')
```